rodan/jobs/helloworld/helloworld.py

```diff
@@ -10,10 +10,20 @@ class HelloWorld(RodanTask):
     interactive = False
 
     input_port_types = (
-        {'name': 'Text input', 'minimum': 0, 'maximum': 1, 'resource_types': ['text/plain']},
+        {
+            'name': 'Text input', 
+            'minimum': 0, 
+            'maximum': 1, 
+            'resource_types': ['text/plain']
+        },
     )
     output_port_types = (
-        {'name': 'Text output', 'minimum': 1, 'maximum': 1, 'resource_types': ['text/plain']},
+        {
+            'name': 'Text output', 
+            'minimum': 1, 
+            'maximum': 1, 
+            'resource_types': ['text/plain']
+        },
```

Comment:
🚫 Please remove these reformatting changes from this PR. These changes are unrelated and have a higher chance of conflicts when we merge with DDMAL later on. 

I know they are causing linter errors. I'll discuss with DDMAL at a later time on a dedicated effort to reformat the code.